release-content/0.15/release-notes/14920_Add_cached_run_system_API.md

```diff
@@ -1,4 +1,28 @@
-<!-- Add cached `run_system` API -->
-<!-- https://github.com/bevyengine/bevy/pull/14920 -->
+# Cached one-shot systems
 
-<!-- TODO -->
+**Bevy 0.15** introduces a new "cached" API for running one-shot systems:
+
+```rust
+// Run a system without manually registering it first:
+commands.run_system_cached(foo);
+commands.run_system_cached_with(bar, 42);
+
+// Reuse a registered system automatically:
+let id1 = world.register_system_cached(quux);
+let id2 = world.register_system_cached(quux);
+assert_eq!(id1, id2);
+```
+
+To better understand why this new API is useful, let's take a look at what systems really are behind the curtains:
```

Comment:
IMO we should show the existing API to demonstrate how much nicer this is, and *then* get into the weeds about the technical details.

Reply:
Made the first code block compare old vs. new API.